WordPressVIPMinimum/Tests/Filters/AlwaysReturnUnitTest.inc

```diff
@@ -77,32 +77,68 @@ if ( ! class_exists( 'another_good_example_class' ) ) {
 	}
 }
 
-add_filter( 'good_example_closure', function() {
+add_filter( 'good_example_closure', function() { // Ok.
 	if ( 1 === 0 ) {
 		return 'Hello';
 	} else {
 		return 'Hey';
 	}
 } );
 
-add_filter( 'bad_example_closure', function() {
+add_filter( 'bad_example_closure', function() { // Error.
 	if ( 1 === 0 ) {
 		return 'hey';
 	}
 	// Missing else return;
 } );
 
-add_filter( 'another_bad_example_closure', function() {
+add_filter( 'another_bad_example_closure', function() { // Error.
 	return;
 } );
 
 if ( 1 === 1 ) {
-	add_filter( 'another_good_example_closure', function () {
+	add_filter( 'another_good_example_closure', function () { // Ok.
 		return 'hello';
 	} );
 }
 
-add_filter( 'bad_example_closure_with_no_returns', function( $input ) {
+add_filter( 'bad_example_closure_with_no_returns', function( $input ) { // Error.
+	$input = $input . ' hello!';
+
+	// But we never return _anywhere_
+}, 10, 2 );
+
+function good_example_arg_pass_by_ref( &$test ) { // Ok.
+	if ( 1 === 0  ) {
+		if ( 1 === 1 ) {
+			return 'ahoj';
+		} else {
+			return $test . 'hello';
+		}
+	}
+	// Missing universal return.
+}
+add_filter( 'good_example_filter', 'good_example_arg_pass_by_ref' );
+
+add_filter( 'good_anonymous_example_arg_pass_by_ref', function( &$input ) { // Ok.
+	$input = $input . ' hello!';
+
+	// But we never return _anywhere_
+}, 10, 2 );
+
+function bad_example_arg( $test ) { // Error.
+	if ( 1 === 0  ) {
+		if ( 1 === 1 ) {
+			return 'ahoj';
+		} else {
+			return $test . 'hello';
+		}
+	}
+	// Missing universal return.
+}
+add_filter( 'bad_example_filter', 'bad_example_arg' );
+
+add_filter( 'bad_anonymous_example_arg', function( $input ) { // Error.
```

Comment:
How is this different from `bad_example_closure_with_no_returns` on line 105?

Reply:
Whoops!  I'm not sure what I was thinking here...removing it now.  Nice catch!